src/org/traccar/protocol/Jt600ProtocolDecoder.java

```diff
@@ -282,4 +303,18 @@ protected Object decode(
         return null;
     }
 
+    private byte checkSum(String sentence) {
```

Comment:
This will require wrapping sub string into `ByteBuffer`. I think it's over-engineering. Maybe it's worth to add an `checkSum(String)` method to the utility class?